backend/pkg/api/enums/enums.go

```diff
@@ -16,6 +16,54 @@ func IsInvalidEnum(e Enum) bool {
 	return e.Int() == -1
 }
 
+type SearchType int
+
+// all possible search types
+const (
+	Name SearchType = iota // default
+	Integer
+	EthereumAddress
+	WithdrawalCredential
+	EnsName
+	NonEmpty
+	Graffiti
+	Cursor
+	Email
+	Password
+	EmailUserToken
+	JsonContentType
+	// Validator Dashboard
+	ValidatorDashboardPublicId
+	ValidatorPublicKeyWithPrefix
+	ValidatorPublicKey
```

Comment:
Since const namespace is package-scoped, pls prefix these enums with `SearchType...`, like with the other enums.